src/Tools/IdeCoreBenchmarks/CloudCache/IdeCoreBenchmarksCloudCacheServiceProvider.cs

```diff
@@ -25,17 +25,7 @@ public IdeCoreBenchmarksCloudCacheServiceProvider()
         public AbstractPersistentStorageService Create(IPersistentStorageConfiguration configuration)
         {
             return new MockCloudCachePersistentStorageService(
-                configuration, @"C:\github\roslyn", cs =>
-                {
-                    if (cs is IAsyncDisposable asyncDisposable)
-                    {
-                        asyncDisposable.DisposeAsync().AsTask().Wait();
-                    }
-                    else if (cs is IDisposable disposable)
-                    {
-                        disposable.Dispose();
-                    }
-                });
+                configuration, @"C:\github\roslyn");
```

Comment:
have gotten confirmation that we only need to support .Dispose for ICacheService proxies, so this all became much simpler.